doc/contributing.rst

```diff
@@ -130,20 +148,25 @@ dependencies as described in our :doc:`parallel backend guide <parallel>`.
 Updating documentation
 ======================
 
-Update ``doc/api.rst`` and ``doc/whats_new.rst`` as appropriate.
+When you update the documentation, it is recommended to build it locally to
+check whether the documentation renders correctly in HTML.
+
+Certain documentation files require frequent updates. These are::
+
+    * ``doc/api.rst`` if you added a new function.
+    * ``doc/whats_new.rst`` to document the fix or change so you can be
+      credited on the next release.
+
+Please update these documents at the end to avoid rebase conflicts with other
+pull requests.
 
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The documentation can be built using sphinx. For that, please additionally
 install the following::
 
-    $ pip install matplotlib sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme pillow mpi4py joblib psutil nbsphinx
-
-If you are using a newer version of pip, you may be prompted to use the flag
-``--use-feature=2020-resolver``. If this happens, please add it as recommended::
```

Comment:
This option is not in all `pip` versions ... if folks get this error, they should just do what the error message says.